doc/source/features/pxc-strict-mode.rst

```diff
@@ -368,8 +376,9 @@ the following happens:
 
  At runtime, any CTAS operation is denied and an error is logged.
 
-.. note:: CTAS operations for temporary tables are permitted
-   even in strict mode.
+.. note::
+
+   CTAS operations for temporary tables are forbidden.
 
```

Comment:
Given that behavior has changed and there is no separate dedicated to CTAS to capture it may be we can add another NOTE section and comment about new behavior here. Note: This behavior is of use only if user is trying to exercise CTAS in disabled mode.

CTAS has SOURCE and DESTINATION tables.

_[ideally user should see this note only when user uses 8.0 manual]_

- Starting PXC-8.0, CTAS replicates as TOI (vs write-set or binlog based replication as it use to be till PXC-5.7)
- SOURCE table(s) should be present on all cluster node. Each cluster node executes CTAS as an independent statement through TOI flow.

- MyISAM tables will be created and loaded even if wsrep_replicate_myisam=1. (Till PXC-5.7 if wsrep_replicate_myisam=0 MyISAM tables were not created as it involved loading too). (PXC doesn;'t recommend use of MyISAM tables and engine. Support for MyISAM is experimental and may be removed in the future release).

- Avoid using a temporary table as source table as a temporary table is not present (or replicated) on all the cluster nodes.

- If node-1 has a temporary and non-temporary table with the same name, CTAS on node-1 will use temporary and CTAS on node-2 will use non-temporary. This may create data level inconsistency.

Reply:
Ready for review